src/codegen/utilities/enum_helpers.py

```diff
@@ -0,0 +1,283 @@
+import logging
```

Comment:
This branch doesn't work on my machine:

```
PS C:\dev\nidaqmx-python> poetry run python -m codegen --dest .\generated\nidaqmx
[    79] ERROR main: Failed to generate
Traceback (most recent call last):
  File "C:\Dev\nidaqmx-python\src\codegen\__main__.py", line 41, in main
    generator.generate(dest)
  File "C:\Dev\nidaqmx-python\src\codegen\generator.py", line 51, in generate
    _generate_file(codegen_metadata,"constants.mako", dest / "constants.py")
  File "C:\Dev\nidaqmx-python\src\codegen\generator.py", line 34, in _generate_file
    f.write(template.render(data=metadata))
  File "C:\Users\bkeryan\AppData\Local\pypoetry\Cache\virtualenvs\nidaqmx-H39Fc4kQ-py3.9\lib\site-packages\mako\template.py", line 439, in render
    return runtime._render(self, self.callable_, args, data)
  File "C:\Users\bkeryan\AppData\Local\pypoetry\Cache\virtualenvs\nidaqmx-H39Fc4kQ-py3.9\lib\site-packages\mako\runtime.py", line 874, in _render
    _render_context(
  File "C:\Users\bkeryan\AppData\Local\pypoetry\Cache\virtualenvs\nidaqmx-H39Fc4kQ-py3.9\lib\site-packages\mako\runtime.py", line 916, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "C:\Users\bkeryan\AppData\Local\pypoetry\Cache\virtualenvs\nidaqmx-H39Fc4kQ-py3.9\lib\site-packages\mako\runtime.py", line 943, in _exec_template
    callable_(context, *args, **kwargs)
  File "C__Dev_nidaqmx_python_src_codegen_templates_constants_mako", line 22, in render_body
ModuleNotFoundError: No module named 'utilities'
```

The `utilities` submodule needs an `__init__.py`.

Reply:
Already added in `master `branch